templates/kubernetes/terraform/modules/kubernetes/variables.tf

```diff
@@ -50,3 +54,13 @@ variable "application_policy_list" {
   type        = list
   default     = []
 }
+
+variable "vpn_server_address" {
+  description = "VPN server address"
+  type        = string
+}
+
+variable "vpn_client_publickeys" {
+  type        = list
```

Comment:
Please change this to list of tuple of 3 strings

Reply:
done with list(tuple([string, string, string]))